docs/manage-fileset-metadata-using-gravitino.md

```diff
@@ -27,6 +27,15 @@ To use fileset, please make sure that:
  - Gravitino server has started, and the host and port is [http://localhost:8090](http://localhost:8090).
  - A metalake has been created.
 
+## Catalog properties
+
+The following are the common properties for all fileset catalogs:
+
+| Property Name       | Description                                                                                                                             | Default Value | Required | Since Version |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `cloud.name`        | The property to specify the cloud that the catalog is running on. The valid values are `aws`, `azure`, `gcp`, `on_premise` and `other`. | (none)        | No       | 0.6.0         |
+| `cloud.region-code` | The property to specify the region code of the cloud that the caatlog is running on.                                                    | (none)        | No       | 0.6.0         |
```

Comment:
There's a chapter in `gravitino-server-config.md` describing about the common catalog properties, maybe you should combine them together.

<img width="1430" alt="image" src="https://github.com/datastrato/gravitino/assets/850797/62095f49-130f-40f3-86c3-978851e83ac7">

Reply:
Maybe you can also move to here.

Reply:
fixed